package.json

```diff
@@ -80,11 +80,11 @@
         "playground"
     ],
     "engines": {
-        "node": ">=10"
+        "node": ">=16.20.2"
```

Comment:
Our own unit tests fail on all versions up to and including v16.0.0. Using v16.20.2 because that's what we're testing against in SFCI.

Reply:
Removed for now since we should really make this update as a breaking change.

Reply:
I'd be surprised if we work with v10. Most of our deps have their own `engines.node` set to much newer versions. If we're strict about it, the lowest version of node that satisfies all of our dependencies' `engines.node` is v18.18.0!